queue_services/entity-emailer/src/entity_emailer/email_processors/restoration_notification.py

```diff
@@ -0,0 +1,174 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for Restoration Application notifications."""
+from __future__ import annotations
+
+import base64
+import re
+from http import HTTPStatus
+
+import requests
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Environment, FileSystemLoader
+from legal_api.models import Business, CorpType, Filing
+
+from entity_emailer.email_processors import get_filing_info
+
+
+def _get_pdfs(
```

Comment:
Difference in attachments in PAID and COMPLETED status.

![image](https://user-images.githubusercontent.com/64869009/229203036-e7fa1e95-1d58-4c41-b9c8-49cbcd479d5c.png)

Reply:
Done, please see unit tests